website/package.json

```
@@ -14,23 +14,24 @@
     "start": "docusaurus start",
     "build": "docusaurus build && yarn run update-redirect ./build/_redirects ./versions.json",
     "build:fast": "PREVIEW_DEPLOY=true yarn run build",
+    "tsc": "npx tsc --noEmit",
     "swizzle": "docusaurus swizzle",
     "deploy": "docusaurus deploy",
     "serve": "docusaurus serve",
     "clear": "docusaurus clear",
     "test": "yarn build",
     "version:cut": "docusaurus docs:version",
-    "format:source": "prettier --write {{core,src}/**/*.js,*.js}",
+    "format:source": "prettier --write {{core,src}/**/*.js,*.js,/**/*.jsx,*.jsx,/**/*.ts,*.ts,/**/*.tsx,*.tsx}",
     "format:markdown": "prettier --write ../docs/*.md && prettier --write {versioned_docs/**/*.md,blog/*.md}",
     "format:style": "prettier --write src/**/*.{scss,css}",
     "format:examples": "eslint-examples-js --fix && eslint-examples-tsx --fix",
     "prettier": "yarn format:source && yarn format:markdown && yarn format:style",
-    "lint": "eslint ../docs/** blog/** core/** src/**/*.js ./*.js",
+    "lint": "eslint ../docs/** blog/** core/** src/**/*.tsx ./*.ts",
     "lint:examples": "eslint-examples-js && eslint-examples-tsx && tsc-examples",
     "lint:versioned": "eslint versioned_docs/**",
     "lint:markdown": "remark ../docs --quiet -r .remarkrc.mjs",
     "lint:markdown:versioned": "remark ./versioned_docs --quiet -r .remarkrc.mjs",
-    "lint:format": "prettier --check {{core,src}/**/*.js,*.js} ../docs/*.md {versioned_docs/**/*.md,blog/*.md} src/**/*.{scss,css}",
+    "lint:format": "prettier --check {{core,src}/**/*.js,*.js,/**/*.jsx,*.jsx,/**/*.ts,*.ts,/**/*.tsx,*.tsx} ../docs/*.md {versioned_docs/**/*.md,blog/*.md} src/**/*.{scss,css}",
```

```suggestion
    "lint:format": "prettier --check {core,src}/**/*.tsx core/**/*.tsx ../docs/*.md {versioned_docs/**/*.md,blog/*.md} src/**/*.{scss,css}",
```

Same issue as above.